.gitignore

```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+.idea
```

Comment:
I use webstorm and it adds a `.idea` folder automatically that should not be added to git.